docs/iam/access-management-plugin.md

```diff
@@ -0,0 +1,159 @@
+# Access Management Plugin Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+MinIO now includes support for using an Access Management Plugin. This is to allow object storage access control to be managed externally via a webhook.
+
+When configured, MinIO sends request and credential details for every API call to an external HTTP(S) endpoint and expects an allow/deny response. MinIO is thus able to delegate access management to an external system, and users are able to use a custom solution instead of S3 standard IAM policies.
+
+CAVEAT: There will be an increased latency due to a request to the external plugin in every request to MinIO.
```

Comment:
`There may be noticeable latency for latency sensitive workloads due to a request to the external plugin upon every authenticated request to MinIO. We strongly advise our users to setup their infrastructure accordingly to their latency needs and performance`

Perhaps a better way to say it.